### PR TITLE
Update ocis-web to 0.2.6

### PR DESCRIPTION
**Pull request by LukasHirt**

### CODE REVIEW

File: changelog/unreleased/add-eos copy.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Enhancement: Update ocis-web to 0.2.6
```
Comment (kulmann) [inline changelog/unreleased/add-eos copy.md:1]: rename this file

Comment (kulmann) [reply changelog/unreleased/add-eos copy.md:1]: version is not 0.2.6, but 0.26.0

File: changelog/unreleased/add-eos copy.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+Tags: ocis-phoenix
+
+We've updated ocis-web to the version 0.2.6.
```
Comment (kulmann) [inline changelog/unreleased/add-eos copy.md:5]: adapt version here as well

File: changelog/unreleased/add-eos copy.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Enhancement: Update ocis-web to 0.2.6
+
+Tags: ocis-phoenix
```
Comment (kulmann) [inline changelog/unreleased/add-eos copy.md:3]: tag for this is supposed to be `web`

**Review by kulmann — CHANGES_REQUESTED** [review]
changelog is a bit off. maybe copy an already existing phoenix update changelog and just update the version and links

**Comment by LukasHirt** [comment]
Dang it. Fixed the changelog.

**Review by kulmann — APPROVED** [review]

**Comment by phil-davis** [comment]
There are lots of UI test fails. The latest `.drone.star` in the Phoenix repo has some different environment variables...
e.g. https://github.com/owncloud/phoenix/commit/eb1361541cffb51278c01ccd0ec9c3be2ccdc03b sorted out a lot of environment variables for the OCIS server that runs during the Phoenix CI. But I see that some of the new/changed things there are actually not mentioned in `.drone.star` of OCIS. Maybe there is some env var magic that is needed here to get things working?

**Comment by kulmann** [comment]
> There are lots of UI test fails. The latest `.drone.star` in the Phoenix repo has some different environment variables...
> e.g. [owncloud/phoenix@eb13615](https://github.com/owncloud/phoenix/commit/eb1361541cffb51278c01ccd0ec9c3be2ccdc03b) sorted out a lot of environment variables for the OCIS server that runs during the Phoenix CI. But I see that some of the new/changed things there are actually not mentioned in `.drone.star` of OCIS. Maybe there is some env var magic that is needed here to get things working?

It looks like the assets were not generated correctly... I just pushed a commit with re-generated assets. Let's see what CI thinks about that.

**Comment by kulmann** [comment]
Web UI tests succeeded now. 👍  waiting for the remaining api tests..

**Comment by phil-davis** [comment]
It's all green. Great stuff, merging.

**Pull request merged**